spec/requests/iiif/auth/v2/probe_service_spec.rb

```diff
@@ -91,6 +92,19 @@
                                                   "status" => 200
                                                 })
       end
+
+      context 'when uri without druid: prefix' do
+        let(:stacks_uri) { "https://stacks-uat.stanford.edu/file/#{id}/#{URI.encode_uri_component(file_name)}" }
+
+        it 'returns a success response' do
+          expect(response).to have_http_status :ok
+          expect(response.parsed_body).to include({
+                                                    "@context" => "http://iiif.io/api/auth/2/context.json",
+                                                    "type" => "AuthProbeResult2",
+                                                    "status" => 200
+                                                  })
+        end
+      end
```

Comment:
note that due to our mocking, this test is not quite comprehensive enough, hence the testing below of the actual parsing method